src/App.js

```diff
@@ -1,12 +1,70 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import './App.css';
+import PokemonCard from "./PokemonCard";
+import axios from "axios";
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokenames, setPokenames] = useState(null);
+
+     const [offset, setOffset] = useState(0);
+    // useEffect -> api pokemon -> [{ name: "bulbasaur"}, { name: "ivysaur"}]
+    useEffect(() => {
+        setPokenames(null) //er moet een nulwaarde meegegeven worden aan setPokenames
+        console.log("fetch names here");
+
+        async function fetchNames() {
+            const response = await axios.get(
+                `https://pokeapi.co/api/v2/pokemon?limit=20&offset=${offset}`
+            );
+            console.log(response.data.results);
+            setPokenames(response.data);
+        }
+        fetchNames();
+    }, [offset]);
+
+    console.log(pokenames);
+if (!pokenames) {
+    return <h3>Loading</h3>
```

Comment:
Als er initieel een error is, dan blijft deze altijd zo staan. Het liefst zou ik in de useEffect willen zien dat een state `loading` op true wordt gezet. Dan wordt het wat meer expliciet